java/openmldb-common/src/main/java/com/_4paradigm/openmldb/common/codec/ClassicRowBuilder.java

```diff
@@ -205,6 +205,9 @@ public boolean appendTimestamp(Timestamp val) {
         if (!check(DataType.kTimestamp)) {
             return false;
         }
+        if (val == null) {
+            return appendNULL();
```

Comment:
Not all jdbc driver can do this `setXX(1,null)->setNull(1)`, it's a good feature, but we should add some desc about it in java sdk doc?